pkg/query-service/utils/format.go

```diff
@@ -143,15 +143,19 @@ func ValidateAndCastValue(v interface{}, dataType v3.AttributeKeyDataType) (inte
 
 // ClickHouseFormattedValue formats the value to be used in clickhouse query
 func ClickHouseFormattedValue(v interface{}) string {
+	// if it's pointer convert it to a value
+	v = getPointerValue(v)
```

Comment:
This functionality is not being used as of now with the new changes ? Not sure if we are going to use it in the future.

Should we keep it or remove ?  @srikanthccv  @makeavish

Reply:
It doesn't hurt to have it